README.md

```diff
@@ -13,20 +13,30 @@ codebase locally and symlink this plugin to that repo. To do so:
 
 1. Clone the build repo and install dependencies
 
-```sh
-git clone git@github.com:netlify/build.git
+    ```sh
+    git clone git@github.com:netlify/build.git
```

Comment:
Personally I'd use the `https` URL here, but I guess it's probably intentional.

Reply:
for non-windows ssh is more common, because people will have their SSH key unlocked via the ssh-agent